index.js

```diff
@@ -250,7 +279,9 @@ function parse(input, options) {
 		return ret;
 	}
 
-	return (options.sort === true ? Object.keys(ret).sort() : Object.keys(ret).sort(options.sort)).reduce((result, key) => {
```

Comment:
Too much logic in the `return` statement makes it hard to understand what's actually returned from the function.